packages/atlas/src/components/AllNftSection/AllNftSection.tsx

```diff
@@ -0,0 +1,183 @@
+    label: 'Not for sale',
+  },
+]
+
+const OTHER: FilterButtonOption[] = [
+  { label: 'Paid promotional material', selected: false, applied: false, value: 'promotional' },
+  { label: 'Mature content rating', selected: false, applied: false, value: 'mature' },
+]
+
+const FILTERS: SectionFilter[] = [
+  {
+    name: 'price',
+    type: 'range',
+    label: 'Price',
+    icon: <SvgActionSettings />,
+    range: { min: undefined, max: undefined },
+  },
+  {
+    name: 'status',
+    label: 'Status',
+    icon: <SvgActionAuction />,
+    type: 'checkbox',
+    options: NFT_STATUSES,
+  },
+  { name: 'other', type: 'checkbox', options: OTHER, label: 'Other', icon: <SvgActionSettings /> },
+]
+
+const LIMIT = 12
+const LG_LIMIT = 30
+
+export const AllNftSection = () => {
+  const [filters, setFilters] = useState<SectionFilter[]>(FILTERS)
+  const [hasAppliedFilters, setHasAppliedFilters] = useState(false)
+  const [order, setOrder] = useState<OwnedNftOrderByInput>(OwnedNftOrderByInput.CreatedAtDesc)
+  const smMatch = useMediaMatch('sm')
+  const lgMatch = useMediaMatch('lg')
+  const limit = lgMatch ? LG_LIMIT : LIMIT
+  const mappedFilters = useMemo((): OwnedNftWhereInput => {
+    const mappedStatus =
+      filters
+        .find((filter) => filter.name === 'status')
+        ?.options?.filter((option) => option.applied)
+        .map((option) => option.value) ?? []
+    const otherFilters = filters.find((filter) => filter.name === 'other')
+    const isMature = otherFilters?.options?.some((option) => option.value === 'mature' && option.applied)
+    const isPromotional = otherFilters?.options?.some((option) => option.value === 'promotional' && option.applied)
+
+    const priceFilter = filters.find((filter) => filter.name === 'price')
+    const minPrice = priceFilter?.range?.appliedMin
+    const maxPrice = priceFilter?.range?.appliedMax
+
+    setHasAppliedFilters(Boolean(minPrice || maxPrice || isPromotional || isMature || mappedStatus.length))
+
+    return {
+      lastSalePrice_gte: minPrice ? String(minPrice) : undefined,
+      lastSalePrice_lte: maxPrice ? String(maxPrice) : undefined,
+      ...(mappedStatus.length
+        ? {
+            transactionalStatus: {
+              isTypeOf_in: mappedStatus,
+            },
+          }
+        : {}),
+      video: {
+        isExcluded_eq: isMature,
+        hasMarketing_eq: isPromotional,
+      },
+    }
+  }, [filters])
+
+  const { nfts, loading, totalCount, fetchMore, pageInfo } = useNftsConnection({
+    where: mappedFilters,
+    orderBy: order,
+    first: limit,
+  })
+  const [isLoading, setIsLoading] = useState(false)
+
+  const placeholderItems = loading || isLoading ? createPlaceholderData(limit) : []
+  const nftsWithPlaceholders = [...(nfts || []), ...placeholderItems]
+  return (
+    <Section
+      headerProps={{
+        onApplyFilters: setFilters,
+        start: {
+          type: 'title',
+          title: 'All NFTs',
+          nodeEnd: totalCount ? (
```

Comment:
According to the design, we should show 0 as well: 
![image](https://user-images.githubusercontent.com/51168865/233059669-7784c08d-6dfc-4d0b-a839-bd4a5664fd6d.png)